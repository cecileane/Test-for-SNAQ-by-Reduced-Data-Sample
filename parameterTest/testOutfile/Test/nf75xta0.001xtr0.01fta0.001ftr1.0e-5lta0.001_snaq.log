optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 0.001.
rootname for files: nf75xta0.001xtr0.01fta0.001ftr1.0e-5lta0.001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 13751
seed: 13751 for run 1
Sat Oct  1 17:37:51 2016

 BEGIN SNaQ for run 1, seed 13751 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 146 (failures: 75) with -loglik=158.13875 and ht_min=[0.32676,0.04619,1.10715,0.29504,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 52		 52		 3	 |	 0		 0		 --
mvtarget	 39		 39		 0	 |	 0		 0		 --
chdir		 8		 8		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 44		 27		 1	 |	 1		 1		 --
Total		 146		 128		 5	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 158.13874671890972
(3,(1,(2,#H9:::0.3267586290371994):10.0):1.1071491853431152,((6,5):0.2950365616136557,(4)#H9:::0.6732413709628007):0.04619057551773972);
---------------------
seed: 55418 for run 2
Sat Oct  1 17:37:59 2016

 BEGIN SNaQ for run 2, seed 55418 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 96 (failures: 75) with -loglik=127.77901 and ht_min=[0.45162,0.52691,0.17558,2.24221,9.92977,0.00614,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 23		 23		 1	 |	 0		 0		 --
mvtarget	 27		 27		 1	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 39		 27		 1	 |	 1		 1		 --
Total		 96		 84		 5	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 127.77901359261531
(((6,(5,#H7:0.17558196165319714::0.45161587269026415):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0::0.5483841273097358);
---------------------

Sat Oct  1 17:38:06 2016
MaxNet is (((6,(5,#H7:0.17558196165319714::0.45161587269026415):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0::0.5483841273097358); 
with -loglik 127.77901359261531
