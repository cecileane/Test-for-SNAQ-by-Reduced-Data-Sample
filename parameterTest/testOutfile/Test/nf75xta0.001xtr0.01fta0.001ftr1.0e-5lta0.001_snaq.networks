(((6,(5,#H7:0.17558196165319714::0.45161587269026415):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0::0.5483841273097358);, with -loglik 127.77901359261531 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(4,3,((5,(#H7:::0.2640323168389549,6):9.880860010754445):0.7016865094498356,((1)#H7:::0.7359676831610451,2):1.5234471686720277):0.003914719871784766);, with -loglik 179.8457843544242
(4,3,((5,(6)#H7:::0.7122467424862455):1.5332188943435987,((#H7:::0.28775325751375447,1):2.8738597276282,2):0.907147733824452):0.012309426640602097);, with -loglik 180.13768684931654
(4,3,((5)#H7:::0.5718949336640651,(((6,#H7:::0.4281050663359349):4.85907284400604,1):0.01898810754539399,2):0.36289643517180104):0.0);, with -loglik 300.00309207711575
(4,3,((5,((1,#H7:::0.4809501010253163):6.7481029524999245,6):0.11637813592968672):0.15242961658636697,(2)#H7:::0.5190498989746837):0.14309960302773697);, with -loglik 315.1398911547373
