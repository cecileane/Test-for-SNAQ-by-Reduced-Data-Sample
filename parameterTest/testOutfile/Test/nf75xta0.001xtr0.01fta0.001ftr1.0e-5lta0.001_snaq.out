(((6,(5,#H7:0.17558196165319714::0.45161587269026415):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0::0.5483841273097358); -Ploglik = 127.77901359261531
 Dendroscope: (((6,(5,#H7:0.17558196165319714):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0);
 Elapsed time: 15.464402267 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((6,(5,#H7:0.17558196165319714::0.45161587269026415):9.929770608045578):2.242205414083886,1):0.5269098557199184,2,((4,3):0.006138496193509352)#H7:0.0::0.5483841273097358);, with -loglik 127.77901359261531
 (3,(1,(2,#H9:::0.3267586290371994):10.0):1.1071491853431152,((6,5):0.2950365616136557,(4)#H9:::0.6732413709628007):0.04619057551773972);, with -loglik 158.13874671890972
-------
